### PR TITLE
Skip activity steps when there is no activity

### PR DESCRIPTION
**Pull request by justinvdm**
@nathanbegbie ready for review (but depends on #359)

### CODE REVIEW

File: src/reducers/__tests__/callNotes-test.js [hunk 1]
```diff
@@ -85,6 +85,35 @@ describe('reducers/callNotes', () => {
...
+        constants.V2_STEP_RATING,
+        constants.V2_STEP_OBJECTIVE_ACHIEVED,
+      ])) {
+        let steps = createStack([
+          createRoute('A'),
+          createRoute(step),
+          createRoute('B'),
+        ]);
+
+        steps = jumpToIndex(steps, 0);
```
Comment (nathanbegbie) [inline src/reducers/__tests__/callNotes-test.js]: I think the step here should be `2` not `0`

File: src/reducers/__tests__/callNotes-test.js [hunk 2]
```diff
@@ -85,6 +85,35 @@ describe('reducers/callNotes', () => {
...
+        const { callNote: state } = fakeState({
+          callNote: {
+            steps,
+            callNote: fakeCallNoteV2({
+              activity: null,
+            }),
+          },
+        });
+
+        expect(reduce(state, actions.v2StepBack()))
```
Comment (nathanbegbie) [inline src/reducers/__tests__/callNotes-test.js]: I think this should be `v2StepNext`

File: src/reducers/__tests__/callNotes-test.js [hunk 3]
```diff
@@ -85,6 +85,35 @@ describe('reducers/callNotes', () => {
...
+          .toEqual(jasmine.objectContaining({
+            steps: jumpToIndex(steps, 0),
```
Comment (nathanbegbie) [inline src/reducers/__tests__/callNotes-test.js]: . . . and this should be `2`?

File: src/reducers/__tests__/callNotes-test.js (hunk 2 above)
Comment (justinvdm) [inline src/reducers/__tests__/callNotes-test.js]: Yep, sorry, thanks

**Comment by nathanbegbie** [comment]
:+1:

**Pull request merged**